content/en/docs/collector/scaling.md

```diff
@@ -382,3 +382,99 @@ service:
       exporters:
         - loadbalancing
 ```
+
+### The Single-Writer Principle
+
+The Single-Writer Principle refers to employing a single logical writer for a
+particular resource. When scaling collectors horizontally in a system it's
+important to properly distinguish between targets using unique identities.
+
+##### Tail-based Sampling
+
+Partial or incomplete traces may be consequential for an implementation of
+tail-sampling, as the goal is to capture all or most of the spans within the
+trace in order to inform sampling decisions. When using the target allocator to
```

Comment:
the target allocator is not related to the tail-based sampling, this seems out of place here

Reply:
Does it make sense to mention it at all here? It's relevant but I am a bit unsure if it actually needs mentioned in a document on collector scaling